src/views/List.jsx

```diff
@@ -1,13 +1,32 @@
+import { useState, useEffect } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchItem, setSearchItem] = useState('');
+	const [filteredItems, setFilteredItems] = useState(data);
+
+	useEffect(() => {
+		setFilteredItems(
+			data.filter((item) =>
+				item.name.toLowerCase().includes(searchItem.toLowerCase()),
+			),
+		);
+	}, [searchItem, data]);
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<form>
+				<label htmlFor="filter_items">Filter items </label>
+				<input
+					type="search"
```

Comment:
Really nice that you used type search which comes with a clear functionality instead of adding your own "x" icon✅✅